diff --git a/node_modules/dexie-observable/dist/dexie-observable.es.js b/node_modules/dexie-observable/dist/dexie-observable.es.js
index 9993760..66a0cf0 100644
--- a/node_modules/dexie-observable/dist/dexie-observable.es.js
+++ b/node_modules/dexie-observable/dist/dexie-observable.es.js
@@ -566,12 +566,12 @@ function Observable(db) {
             throw new Error("Mixed versions of dexie-observable");
         return; // Addon already active.
     }
-    var NODE_TIMEOUT = 20000, // 20 seconds before local db instances are timed out. This is so that old changes can be deleted when not needed and to garbage collect old _syncNodes objects.
-    HIBERNATE_GRACE_PERIOD = 20000, // 20 seconds
+    var NODE_TIMEOUT = 5000, // 20 seconds before local db instances are timed out. This is so that old changes can be deleted when not needed and to garbage collect old _syncNodes objects.
+    HIBERNATE_GRACE_PERIOD = 5000, // 20 seconds
     // LOCAL_POLL: The time to wait before polling local db for changes and cleaning up old nodes. 
     // Polling for changes is a fallback only needed in certain circomstances (when the onstorage event doesnt reach all listeners - when different browser windows doesnt share the same process)
     LOCAL_POLL = 500, // 500 ms. In real-world there will be this value + the time it takes to poll(). A small value is needed in Workers where we cannot rely on storage event.
-    HEARTBEAT_INTERVAL = NODE_TIMEOUT - 5000;
+    HEARTBEAT_INTERVAL = 1000;
     var localStorage = Observable.localStorageImpl;
     /** class SyncNode
         *
