diff --git a/node_modules/observable-hooks/dist/esm/use-observable-eager-state.js b/node_modules/observable-hooks/dist/esm/use-observable-eager-state.js
index 9299895..f92c97f 100644
--- a/node_modules/observable-hooks/dist/esm/use-observable-eager-state.js
+++ b/node_modules/observable-hooks/dist/esm/use-observable-eager-state.js
@@ -19,7 +19,7 @@ export function useObservableEagerState(state$) {
     var forceUpdate = useForceUpdate();
     var state$Ref = useRef(state$);
     var errorRef = useRef();
-    var isAsyncEmissionRef = useRef(false);
+    var isAsyncEmissionRef = useRef(true);
     var _a = useState(function () {
         var state;
         state$
@@ -53,6 +53,9 @@ export function useObservableEagerState(state$) {
                     // ignore synchronous value
                     // prevent initial re-rendering
                     setState(value);
+                    if (value === state) {
+                      forceUpdate();
+                    }
                 }
             },
             error: function (error) {
diff --git a/node_modules/observable-hooks/dist/esm2015/use-observable-eager-state.js b/node_modules/observable-hooks/dist/esm2015/use-observable-eager-state.js
index 63adb80..48153bc 100644
--- a/node_modules/observable-hooks/dist/esm2015/use-observable-eager-state.js
+++ b/node_modules/observable-hooks/dist/esm2015/use-observable-eager-state.js
@@ -53,6 +53,8 @@ export function useObservableEagerState(state$) {
                     // ignore synchronous value
                     // prevent initial re-rendering
                     setState(value);
+                    forceUpdate();
+                  console.log('hey!')
                 }
             },
             error: error => {
