diff --git a/node_modules/socketcluster-client/node_modules/ws/lib/websocket-server.js b/node_modules/socketcluster-client/node_modules/ws/lib/websocket-server.js
index 72618d0..cbac102 100644
--- a/node_modules/socketcluster-client/node_modules/ws/lib/websocket-server.js
+++ b/node_modules/socketcluster-client/node_modules/ws/lib/websocket-server.js
@@ -1,6 +1,6 @@
 'use strict';
 
-const EventEmitter = require('events');
+const { EventEmitter } = require('events');
 const crypto = require('crypto');
 const http = require('http');
 const url = require('url');
diff --git a/node_modules/socketcluster-client/node_modules/ws/lib/websocket.js b/node_modules/socketcluster-client/node_modules/ws/lib/websocket.js
index 67cd8a4..902d85d 100644
--- a/node_modules/socketcluster-client/node_modules/ws/lib/websocket.js
+++ b/node_modules/socketcluster-client/node_modules/ws/lib/websocket.js
@@ -1,6 +1,6 @@
 'use strict';
 
-const EventEmitter = require('events');
+const { EventEmitter } = require('events');
 const crypto = require('crypto');
 const https = require('https');
 const http = require('http');
